src/components/shortAnswer/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import {
+    Box,
+    Button,
+    Stack,
+    Heading,
+    Input,
+    Flex,
+    InputGroup,
+    InputRightElement,
+    CloseButton,
+    Spacer,
+} from "@chakra-ui/react";
+import { Question } from "pages/admin/dashboard/builder";
+
+export interface ShortAnswerProps {
+    questions: Question[];
+    setQuestions: (newQuestions: Question[]) => void;
+}
+interface ShortAnswerQuestionProps {
+    index: number;
+    value: Question;
+    onChange: (newQuestion: Question, index: number) => void;
+    onDelete: (index: number) => void;
+}
+
+const ShortAnswerQuestion: React.FC<ShortAnswerQuestionProps> = ({
+    index,
+    value,
+    onChange,
+    onDelete,
+}) => {
+    const { question } = value;
+    return (
+        <Flex>
+            <Stack width={"full"} pb={8}>
+                <Heading size="md">Short Answer {index + 1}</Heading>
+                <InputGroup>
+                    <Input
+                        variant="flushed"
+                        placeholder="Text Here"
+                        onChange={(e) => {
+                            onChange({ question: e.target.value }, index);
+                        }}
+                        value={question}
+                    />
+                    <InputRightElement width="4.5rem">
+                        <CloseButton onClick={() => onDelete(index)} />
+                    </InputRightElement>
+                </InputGroup>
+            </Stack>
+        </Flex>
+    );
+};
+
+export const ShortAnswer: React.FC<ShortAnswerProps> = ({
+    questions,
+    setQuestions,
+}) => {
+    const onQuestionsChange = (newQuestion, index) => {
+        const newQuestions = [...questions];
+        newQuestions[index] = newQuestion;
+        setQuestions(newQuestions);
+    };
+
+    const deleteQuestion = (index: number) => {
+        const newQuestions = [...questions];
+        newQuestions.splice(index, 1);
+        setQuestions(newQuestions);
+    };
+
+    const questionFields = (
+        <Box>
+            {questions.map((currentQuestion, i) => (
```

Comment:
I think it will make it easier to support user variables if each content section only had 1 user input

Reply:
Ah that's true, good product thinking